### DISCUSSION TITLE
Frage zur Lizenz

### DISCUSSION THREAD

**Discussion by Marv2023** [brutella/hkknx-public#248] [Q&A] [answered]
Hallo,
ich bin neu hier. Ich habe wenig knx Erfahrung, möchte aber darin und in Verbindung Mit HomeKit besser werden. Hintergrund ist das mein Elternhaus komplett mit KNX gesteuert wird und ich des als Visualisierung via HomeKit steuern möchte.

Wenn ich jetzt mehr als 15KNX Geräte hinzufügen möchte, muss ich ja die Lizenz kaufen. Wenn ich das jedoch versuche, wird immer nur Fehler angezeigt auf der hkknx.local Seite. 

Meine Frage: Wie komme ich zu dir Lizenz und wieviel kostet die mich. 

Des weiteren kenne ich mich leider gar nicht mit dem Raspberry Pi aus. Ich war schon froh dass ich das Img von hkknx draufspielen und die erste 5 Geräte programmieren konnte. Jetzt möchte ich auch Homebridge installieren, weiß aber nicht wie ich das machen soll, weil so wie ich das jetzt mit dem Raspberry Pi Imager verstehe, würde der ja beim Versuch Homebridge zu installieren, hkknx überschreiben. Gibt es da noch eine andere Möglichkeit. Entschuldigung schon einmal für die Unwissenheit.

**Comment by brutella** [comment, marked as answer]
> Wenn ich jetzt mehr als 15KNX Geräte hinzufügen möchte, muss ich ja die Lizenz kaufen. Wenn ich das jedoch versuche, wird immer nur Fehler angezeigt auf der hkknx.local Seite.

Welcher Fehler wird beim Kauf einer Lizenz angezeigt?

Wirf mal einen Blick in das Online Handbuch. Dort findest du wichtige Tipps zu hkknx.
https://hochgatterer.me/de/hkknx/docs/help/

[a comment by an automated account is not shown]
